### PR TITLE
Create Postgres sink guide

### LINKED ISSUE #524: Document the JDBC sink connector
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/6493

### Which part(s) of the docs might be affected or should be updated? And how?

Data delivery -> (two integration guides?) Sink data to MySQL & Sink data to PostgreSQL
Data delivery -> Overview: Describe the JDBC connector, and the possiblity of using it to sink to other databases such as Druid. 

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Add guide on sinking to PostgreSQL

- **Preview**: 
https://pr-604.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/sink-to-postgres/

- **Related code PR**: 

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/524

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/guides/sink-to-postgres.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+---
+ id: sink-to-postgres
+ title: Sink data from RisingWave to PostgreSQL JDBC connector
+ description: Sink data from RisingWave to PostgreSQL with JDBC connector.
+ slug: /sink-to-postgres
+---
+
+In this guide, we will introduce how to sink data from RisingWave to JDBC-available databases using the JDBC sink connector. Using RisingWave, we will be using a load generator and consuming data from a Kafka broker then sinking to a PostgreSQL database. You can test out this process on your own device by using the `postgres-sink` demo in the [`risingwave-demo`](https://github.com/risingwavelabs/risingwave-demo ) respository.
```
Comment (hengm3467) [inline docs/guides/sink-to-postgres.md]: @emile-00 the risingwave-demo was moved to the risingwave repository under https://github.com/risingwavelabs/risingwave/tree/main/integration_tests

File: docs/guides/sink-to-postgres.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+### Sink from RisingWave
+
+Use the following query to sink data from the materialized view to the target table in PostgreSQL. Ensure that the `jdbc_url` is accurate and reflects the PostgreSQL database that you are connecting to. See [`CREATE SINK`](../sql/commands/sql-create-sink.md) for more details.
+
+```sql
+CREATE SINK target_count_postgres_sink FROM target_count WITH (
+    connector = 'jdbc',
+    jdbc.url = 'jdbc:postgresql://postgres:5432/mydb?user=myuser&password=123456',
+    table.name = 'target_count'
```
Comment (neverchanje) [inline docs/guides/sink-to-postgres.md]: `type = 'upsert'` 

This field is missing.

**Comment by neverchanje** [comment]
The rest looks perfect to me!

**Review by hengm3467 — APPROVED** [review]
I made some edits to the draft. And it now looks good to me.

**Pull request merged**